packages/transport-axios/src/AxiosTransport.js

```diff
@@ -0,0 +1,33 @@
+import HttpTransport from '@stellarjs/transport-http';
+
+class AxiosTransport extends HttpTransport {
+  constructor(axios, source, log, requestTimeout, baseUrl) {
```

Comment:
Why it gets axios as a parameter?

Reply:
Requested by @sloops77

Reply:
standard usage of dependency injection in all transports